jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ContentProducer.java

```diff
@@ -104,16 +104,17 @@
      * The returned content is decoded by the interceptor set with {@link #setInterceptor(HttpInput.Interceptor)}
      * or left as-is if no intercept is set.
      * After this call, state can be either of UNREADY or IDLE.
+     *
      * @return the next content that can be read from or null if the implementation does not block
      * and has no available content.
      */
-    Content nextContent();
+    Content.Chunk nextContent();
 
     /**
-     * Free up the content by calling {@link Content#release()} on it
+     * Free up the content by calling {@link Content.Chunk#release()} on it
      * and updating this instance' internal state.
      */
-    void reclaim(Content content);
+    void reclaim(Content.Chunk content);
```

Comment:
Now that this class isn't dealing with `Content`s but `Chunk`s the following method `onContentProducible` should be renamed.